Editor/Window/RegisterComputeInput.cs

```diff
@@ -114,6 +118,49 @@ private async void OnRegisterComputeButtonClicked()
                         Strings.ViewLogsStatusBoxUrlTextButton);
                 }
             }
+            
+            UpdateGUI();
+        }
+
+        private async void CheckCompute()
+        {
+            if (_computeState is ComputeStatus.Registered)
+            {
+                var computes = await _stateManager.ComputeManager.ListCompute(_stateManager.AnywhereFleetId);
+                if (computes.Success)
+                {
+                    var matchingCompute =
+                        computes.ComputeList.FirstOrDefault(compute =>
+                            compute.ComputeName == _stateManager.ComputeName);
+                    if (matchingCompute == null)
+                    {
+                        _computeNameInput.value = _defaultComputeName;
+                        var defaultIpAddress = _defaultIpAddress.Split(".");
+                        for (var i = 0; i < defaultIpAddress.Length; i++)
+                        {
+                            _ipInputs[i].value = defaultIpAddress[i];
+                        }
```

Comment:
I'd recommend writing a utility to do this for you. Perhaps `SetIpInputs(string IpAddress)`

Reply:
Moved 3 uses to the utility